modules/core/client/components/LanguageSwitch.component.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import i18n from './i18n';
+import PropTypes from 'prop-types';
+import { Dropdown, MenuItem } from 'react-bootstrap';
```

Comment:
Does this import Bootstrap v3 or v4 HTML?

We're still using v3 stylesheets so we'd need v3 HTML.

Reply:
:+1: 

I'm in the process of doing that now.

`react-bootstrap` uses [bootstrap v3](https://react-bootstrap.github.io/).

Reply:
moved to #957